routing/restrictions_serialization.hpp

```diff
@@ -35,8 +36,12 @@ struct Restriction
   /// * driving along the restriction is prohibited (No)
   enum class Type
   {
-    No,    // Going according such restriction is prohibited.
-    Only,  // Only going according such restriction is permitted
+    No,        // Going according such restriction is prohibited.
+    Only,      // Only going according such restriction is permitted.
+
+    NoUTurn,   // Restriction about forbidding making uturn from some
+               // Feature_№_N to the same Feature_№_N through some node at this feature.
```

Comment:
`Making U-turn from a feature to the same one according to such restriction is prohibited.`?
https://dictionary.cambridge.org/dictionary/english/u-turn